### ISSUE TITLE
SC file spec

### ISSUE THREAD

**Issue by norman784** [dimitrilozovoy/Voxyc#1]
Hi, I saw your post in gamedev.net, the amount of effort you put in this engine is very impressive, keep improving it.

Now my question I'm interested in the specs of the SC file.

- There's somewhere in your source or docs where you describe it?
- The SC file spec is stable?

Regards

**Comment by dimitrilozovoy** [comment]
FileIO.cpp writes and reads it  That and Scene.cpp and Voxels.cpp.

**Comment by dimitrilozovoy** [comment]
https://github.com/dimitrilozovoy/Voxyc/blob/master/engine/Scene.cpp

This has the save and the load functions.

**Comment by dimitrilozovoy** [comment]
Yes, it's quite stable. I've been developing binary formats for about 5
years now.

**Comment by norman784** [comment]
Thanks, will take a look at them.
